check_pgactivity

```diff
@@ -7420,6 +7420,8 @@ that it cannot access, nor live temp files
  v11: an unprivileged role is possible but must be granted EXECUTE
 on functions pg_ls_dir(text), pg_read_file(text), pg_stat_file(text);
 the same restrictions than on v10 will still apply
+ v12: a user with the role pg_monitor suffices. unprivileged role is not
```

Comment:
I'd personally drop the "unprivileged role is not supported anymore", as it actually wasn't the case before.

Reply:
Done in commit 13324c7

Reply:
Wording improved as per commit d3f3cda